src/collapse/dtrace.rs

```diff
@@ -0,0 +1,203 @@
+use super::Collapse;
+use std::collections::{HashMap, VecDeque};
+use std::io;
+use std::io::prelude::*;
+
+// Most code should not have more then 10 calls in the java stack.
+const JAVA_VEC_SIZE: usize = 10;
```

Comment:
I don't think we even need this -- just use `::new()` :)

Reply:
👍